src/parser/jobs/sam/modules/Ogi.tsx

```diff
@@ -0,0 +1,72 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+export class Ogi extends Analyser {
+	static override handle = 'ogi'
+	@dependency private checklist!: Checklist
+	@dependency private data!: Data
+
+	private OgiReadys = 0
+	private OgisDone = 0
+
+	Ogis = [
+		this.data.actions.OGI_NAMIKIRI.id,
+		this.data.actions.KAESHI_NAMIKIRI.id,
+	]
```

Comment:
Same as kaiten, might want an `ActionArray[]` and then `this.data.matchActionId(OGI_ACTIONS)` in the hook (which internally uses the oneOf stuff). I'd fix variable name casing and make it private tho if not.